### DISCUSSION TITLE
[24/04/19] 백엔드 회의록

### DISCUSSION THREAD

**Discussion by ghrltjdtprbs** [10aeat/10aeat_Backend#6] [백엔드 회의록]
## 회의 내용

**1. 코드 컨벤션 결정**
- #3 

**2. 커밋 메시지 작성 안내**

**3. 백엔드 팀 디스커션 사용 결정**
    - 프론트가 디스커션을 사용할지는 프론트와 회의 후 결정 예정

**4. issue와 project로 일정 관리**
     - project 템플릿 설정 예정 @ghrltjdtprbs 

**5. pr템플릿 설정 예정** @ghrltjdtprbs 

**6. JWT인증 방식 결정은 프론트와 회의 후 결정 예정**
- #4  

----------
## 특이 사항
- @Galmaeki @asd42270 미니 프로젝트 리팩토링 진행 중

**Comment by yerim08** [comment]
저희도 디스커션 사용하기로 했습니당 !

**Reply by ghrltjdtprbs** [reply]
확인했습니다!

**Comment by humanpear** [comment]
JWT인증 방식 Header로 사용하기로 했습니다 :)

**Reply by ghrltjdtprbs** [reply]
확인했습니다!
